plugins/Cargo.toml

```diff
@@ -23,6 +23,7 @@ travis-ci = { repository = "google/tarpc" }
 proc-macro2 = "1.0"
 quote = "1.0"
 syn = { version = "2.0", features = ["full"] }
+proc-macro-warning = "1"
```

Comment:
Just noticed this — are any well-known crates using it? Just want to make sure I do some due diligence before taking on a new (unfamiliar to me) dependency.

Reply:
Not off the top of my head, if you know a way to emit a warning without this I'm happy to refactor.

Reply:
Looking at the crate, I think the magic is just these lines: https://github.com/ggwpez/proc-macro-warning/blob/970809f551eb78ea003006ef4da0c303ede8501d/proc-macro-warning/src/lib.rs#L260-L272

Basically, it just adds a secret line like this:
```rust
#[deprecated(note = "...")]
const __DEPRECATION_NOTICE: () = ();
let _ = __DEPRECATION_NOTICE;
```

Reply:
Ok, want me to remove the dependency and do something like that?

Reply:
Yes, I think I'd prefer that, since looks to be just a small amount of code. Thanks very much!

Reply:
Will do!

Reply:
Done! I like this output better anyways 🙂